robottelo.properties.sample

```diff
@@ -188,3 +188,12 @@ ssh_key=
 # Parameter for deciding whether conduct initial sync or resync
 # 'resync' denotes resync; 'sync' denotes initial sync
 # sync_type='sync'
+
+# [libvirt_host]
```

Comment:
How about we making this [compute_resources] ?
I am not asking to change this, but rather so we discuss and think about it, if it is OK ?

Thinking In future all the other(vmware, rhevm, openstack) CR details also could go below this.